clase 1 19.4/src/main/java/services/GamePlay.java

```diff
@@ -0,0 +1,201 @@
+package services;
+
+import interfaces.IBoardBuilder;
+import interfaces.IFinder;
+import interfaces.IGamePlay;
+import scanner.IScanner;
+import scanner.ScannerClass;
+import scanner.impl.MenuUserDecisionImpl;
+import java.util.*;
+
+public class GamePlay implements IGamePlay {
+
+    private IBoardBuilder boardBuilder;
+    private IFinder finder = new FinderClass();
```

Comment:
looks like this variable should be a final variable because the value never changes, right?
https://www.javatpoint.com/final-keyword